### DISCUSSION TITLE
Copilot Payment Paypal

### DISCUSSION THREAD

**Discussion by chandraawardhana** [community/community#30483] [Copilot Conversations]
Hello,
I have a problem when I make a payment for the copilot. I paid 1 year with $100 and it has been deducted in my PayPal, but I still received a notification that my copilot will end tonight and currently I still can't use my copilot because my trial time is up. Please help. Thank you

**Comment by strugglebak** [comment]
seems only Chinese users got the same problem of Copilot payment...

**Reply by chandraawardhana** [reply]
no I'm from indonesia

**Reply by strugglebak** [reply]
> no I'm from indonesia

I'm sure it must be a big bug, but no staff care about it, weird

**Comment by strugglebak** [comment]
up

**Comment by strugglebak** [comment]
up

**Comment by iamcjwilliams** [comment]
Hi there! The best place to go for  account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket  and they should be able to help you get this sorted :)
